packages/plugin-loyalties-api/src/models/ScoreLogs.ts

```
@@ -138,17 +138,27 @@ export const loadScoreLogClass = (models: IModels, subdomain: string) => {
             totalPointRedeemed: {
               $sum: {
                 $cond: {
-                  if: {$or: [{ $eq: ['$action', 'subtract'] }, { $lt: ['$changeScore', 0] }]},
-                  then: {$abs: '$changeScore'},
+                  if: {
+                    $or: [
+                      { $eq: ["$action", "subtract"] },
+                      { $lt: ["$changeScore", 0] },
+                    ],
+                  },
+                  then: { $abs: "$changeScore" },
```

Biome (1.9.4)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)